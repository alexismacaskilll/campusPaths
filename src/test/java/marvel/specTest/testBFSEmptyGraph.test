# Test the parser on a file that is empty

# Load a graph from empty tsv file
LoadGraph emptyGraph empty.tsv

# Find shortest path from hero1 to hero2
FindPath emptyGraph hero1 hero2