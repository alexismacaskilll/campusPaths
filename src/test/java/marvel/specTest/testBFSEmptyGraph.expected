# Test the parser on a file that is empty

# Load a graph from empty tsv file
loaded graph emptyGraph

# Find shortest path from hero1 to hero2
unknown character hero1
unknown character hero2